spec/spec_helper.rb

```diff
@@ -1,5 +1,14 @@
+require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.